backend/src/main/java/de/laura/project/db/TempTriviaQuestionDB.java

```diff
@@ -17,38 +17,48 @@
 @NoArgsConstructor
 @Builder
 
-@Service
+@Repository
 public class TempTriviaQuestionDB {
 
-    private List<TriviaQuestionSet> triviaQuestionSetList;
+    private List<TriviaQuestionSet> triviaQuestionSetList = new ArrayList<>();
 
 
     public List<TriviaQuestionSet> returnQuestionList(List<TriviaQuestionSet> triviaQuestionSetList) {
-        this.triviaQuestionSetList = triviaQuestionSetList;
         return triviaQuestionSetList;
     }
 
-    public TriviaQuestionSetWithoutAnswer getSingleQuestion(int id) {
+    public TriviaQuestionSetWithoutCorrectAnswer getSingleQuestion(int questionId) {
 
-        List<TriviaQuestionSetWithoutAnswer> triviaQuestionSetListWithoutAnswer = new ArrayList<>();
+        List<TriviaQuestionSetWithoutCorrectAnswer> triviaQuestionSetListWithoutAnswer = new ArrayList<>();
```

Comment:
```suggestion
        List<TriviaQuestionSetWithoutCorrectAnswer> triviaQuestionSetListWithoutCorrectAnswer = new ArrayList<>();
```

maybe consider keeping a consisting naming convention
I am a bit confused that a list contains multiple sets - does one set include multiple questions? or is each question one object? in that case I would suggest naming it `triviaQuestionWithoutCorrectAnswer`